bits/reboot_hook.s

```diff
@@ -83,6 +84,9 @@ koffset_base:    .word kernel_code-jump_to_kernel-12
 .pool
 
 kernel_code:
+
+    ldr sp, =0x080FF000
+
```

Comment:
Looks like it works without that line, but I only tested GBA games cause my TWL_FIRM or corbenik or my A9LH are broken and don't let corbenik load and patch TWL_FIRM... I'll wait for Crystal The Glacion confirm if it works on TWL games/homebrews